doc/JSON_INFO.md

```diff
@@ -1265,8 +1265,8 @@ See also VEHICLE_JSON.md
 "price": 100,                    // Used when bartering with NPCs. For stackable items (ammo, comestibles) this is the price for stack_size charges. Can use string "cent" "USD" or "kUSD".
 "price_post": "1 USD",           // Same as price but represent value post cataclysm. Can use string "cent" "USD" or "kUSD".
 "material": ["COTTON"],          // Material types, can be as many as you want.  See materials.json for possible options
-"cutting": 0,                    // (Optional, default = 0) Cutting damage caused by using it as a melee weapon
-"bashing": -5,                   // (Optional, default = 0) Bashing damage caused by using it as a melee weapon
+"cutting": 0,                    // (Optional, default = 0) Cutting damage caused by using it as a melee weapon.  Cannot be negative?
```

Comment:
```suggestion
"cutting": 0,                    // (Optional, default = 0) Cutting damage caused by using it as a melee weapon.
```
If you don't know for sure whether or not it can be negative, don't put it in the documentation.

Reply:
The documentation previously said it could be negative.  So I asked about it, and a person who ought to know assured me that it definitely could not be negative, and that I should update the doc.

So I am "second-hand-sure."

Reply:
Either you are certain, in which case write it as such, or you aren't, in which case don't write it. Don't write something as if it's unclear in the documentation.

Reply:
fair enough. I know bashing cannot negative because I was told by two members of the dev team and I tested it myself. 

I don't know why that is or how it works, and so I can't say it the same applies to cutting.

So I'll resolve this by adding a period to the end of the note for bashing, leaving bashing at 0, and removing my note on cutting. Thanks Erk.

Reply:
Cutting can also not be negative.